.env.example

```diff
@@ -1,4 +1,7 @@
 PORT=3333
-POSTGRES_USER=postgres
-POSTGRES_HOST=127.0.01
-POSTGRES_DB=hello_world
+PP_KEYS=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_HOST=localhost
+POSTGRES_DB=
+DB_CONNECTION_STRING=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRESS_HOST}:5432/${POSTGRES_DB}
```

Comment:
Sadly with `dotenv` I don't think interpolation works. `direnv` is fine with it, but that needs setting up at the system level.